### ISSUE TITLE
I'm getting loss as NAN values 

### ISSUE THREAD

**Issue by Guttappa1238** [anhtuan85/Pytorch-SSD-from-scratch#2]
Hi,

i'm training single custom class using default parameters when i start training i got following results which loss has  **nan**  values.

![Screenshot from 2021-10-30 16-23-24](https://user-images.githubusercontent.com/45628395/139530116-88668d62-4353-4526-a9a5-4ca640455395.png)

please can anyone explain why i'm getting this?

Thanks advance

**Comment by anhtuan85** [comment]
Have you set num_classes = 1?
Then you should check that the data and labels are correct. 
And check two loss functions smooth_L1_loss and CrossEntropyLoss

**Comment by Guttappa1238** [comment]
I set it correctlty, still getting same NAN values.

Please can you add readme doc that explains how to train custom objects? which will be helpful for others too

**Comment by anhtuan85** [comment]
Hi, 
I'm quite busy right now. I will update the code and documentation in the next 1-2 weeks.
Thank you!

**Comment by Guttappa1238** [comment]
> Hi, I'm quite busy right now. I will update the code and documentation in the next 1-2 weeks. Thank you!

Oh okay no problem.

Thanks for quick response
